themes/digital.gov/src/scss/new/_join-community-form.scss

```diff
@@ -0,0 +1,55 @@
+@use "uswds-core" as *;
+
+// Styles for inline form on the DG7 community pages
+.join-community-form {
+  @include u-margin-bottom(5);
+  @include u-padding-bottom(4);
+  @include u-padding-x(3);
+  @include u-text("light");
+  @include u-radius("sm");
+  @include u-padding-top(2.5);
+  @include u-bg("blue-5");
+  @include u-text("gray-90");
+  @include u-border-left("05", "solid", "blue-20");
+  &__header {
```

Comment:
Can we add empty lines before these selectors to make it easier to read?

Reply:
Yes, good point!